### ISSUE TITLE
Framework Problem

### ISSUE THREAD

**Issue by Ariansdf** [Placenote/PlacenoteSDK-Unity#61]
Hi,
I've tried installing placenote with both methods, but with each one I have some kind of issue with the framework.
As I download unity package, I have the framework folder completely downloaded. (8Mb). However, as I try to build the example project I become the error:

...iOS/Placenote.framework/Placenote' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target.

Moreover, I can't add the framework from the list in Xcode but I still can drag and drop the folder.
I tried every other solution I found on the internet but nothing worked for me.
So I used the SSH key method and cloned the project from GitHub. However, the framework folder is only 61KB. I repeated the process several times but the result was still the same. (Home-brew is also properly installed).

I'd be happy if someone could offer me a solution.
Thanks

**Comment by pmukherj** [comment]
Hey @Ariansdf , 

Make sure to go through the appropriate steps on our documentation page. Especially Step 3 on our instructions in XCode (https://docs.placenote.com/unity/build-the-sample-apps/run-on-an-ios-device). You have to specifically turn off bitcode in XCode to get past the ENABLE_BITCODE error.

**Comment by Ariansdf** [comment]
> Hey @Ariansdf ,
> 
> Make sure to go through the appropriate steps on our documentation page. Especially Step 3 on our instructions in XCode (https://docs.placenote.com/unity/build-the-sample-apps/run-on-an-ios-device). You have to specifically turn off bitcode in XCode to get past the ENABLE_BITCODE error.

Hi,
Thanks for fast answer. Of course I have double checked every step. I even made 3 different Unity Projects but I still get the same error in Xcode.
After I posted here I found out that I get another error also in Unity. (Not sure if I got that the first time)

Exception: Calling TargetGuidByName with name='Unity-iPhone' is deprecated. There are two targets now, call GetUnityMainTargetGuid() - for app or GetUnityFrameworkTargetGuid() - for source/plugins to get Guid instead.
UnityEditor.iOS.Xcode.PBXProject.TargetGuidByName (System.String name) (at /Users/bokken/buildslave/unity/build/External/XcodeAPI/Xcode/PBXProject.cs:172)
PostBuildTrigger.OnPostprocessBuild (UnityEditor.BuildTarget target, System.String pathToBuiltProject) (at Assets/Placenote/Editor/PostBuildTrigger.cs:21)
...

Thanks.
